.github/Dockerfile

```diff
@@ -25,13 +25,21 @@ RUN sed -i '/buffer_with_truncation /d' /bats-core/libexec/bats-core/bats-format
 # Remove possible old build files
 RUN rm -rf cmake && \
 # Build and test FTL
-    bash build.sh "-DSTATIC=${STATIC}" test ${BUILD_OPTS} && \
-# Move FTL binary to root directory
+    bash build.sh "-DSTATIC=${STATIC}" ${BUILD_OPTS} && \
+# Copy FTL binary to root directory
     cd / &&\
-    mv /app/pihole-FTL . && \
+    cp /app/pihole-FTL . && \
 # Create tarball of API docs
     tar -C /app/src/api/docs/content/ -czvf /api-docs.tar.gz .
 
+FROM builder AS tests
+WORKDIR /app
+# Run tests
+# If BUILD_OPTS is set to "clang", we set STATIC="false" as we do in build.sh
+RUN if [ ${BUILD_OPTS} = "clang" ]; then export STATIC="false";fi && \
```

Comment:
That `RUN` will execute under `sh` and not `bash` so it will fail:

https://github.com/pi-hole/FTL/actions/runs/15499949565/job/43645424321#step:3:2362

![image](https://github.com/user-attachments/assets/dd1cd340-3ef3-478c-90f2-d1a6cfdb32c4)

Reply:
Fixed 

![2025-06-08_07-14](https://github.com/user-attachments/assets/de36a5ab-1330-4ccc-8cd1-43c60e9007fe)